compliance_wi_compliance.md

```diff
@@ -1,10 +1,8 @@
 Wisconsin Compliance
 ====================
+M-RETS is the designated tracking system for the state of Wisconsins RPS. In order for certificates to be used for WI Compliance the generator associated with the the certitifcates must have the necessary Wisconsin eligibilities applied to it. Generators seeking "Wisconsin: Bankable: eligiblity must submit the necessary approval documents from the Wisconisn Public Service Commission.
```

Comment:
Edits:
State of Wisconsin's RPS
To use certificates for State of Wisconsin compliance, the generator...

Reply:
@markmrets Please edit this use of the quote.